$OpenBSD: patch-mcop_mt_threads_posix_cc,v 1.1 2019/06/08 15:25:57 jca Exp $

Move header to the appropriate location, ie not in the middle of the
Arts::PosixThreads namespace.

Index: mcop_mt/threads_posix.cc
--- mcop_mt/threads_posix.cc.orig
+++ mcop_mt/threads_posix.cc
@@ -28,6 +28,7 @@
 #ifdef HAVE_LIBPTHREAD
 
 #include <gsl/gslconfig.h>
+#include <gsl/gslcommon.h>
 
 #include <sys/types.h>
 #include <stddef.h>
@@ -332,7 +333,6 @@ static class SetSystemThreads { (public)
 	}
 } initOnStartup;
 
-#include <gsl/gslcommon.h>
 /* -fast- locking for gsl on platforms with unix98 support */
 #if (GSL_HAVE_MUTEXATTR_SETTYPE > 0)
 static void pth_mutex_init (GslMutex *mutex)
